elrond.md

```diff
@@ -961,7 +977,13 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
 ```k
     // extern long long smallIntGetUnsignedArgument(void *context, int32_t id);
     rule <instrs> hostCall("env", "smallIntGetUnsignedArgument", [ i32 .ValTypes ] -> [ i64 .ValTypes ])
-               => #returnIfUInt64(Bytes2Int({ARGS[ARG_IDX]}:>Bytes, BE, Unsigned)) ... </instrs>
+               => #returnIfUInt64(Bytes2Int({ARGS[ARG_IDX]}:>Bytes, BE, Unsigned), "argument out of range") ... </instrs>
```

Comment:
"argument out of range" sounds like you tried to access an argument with index too high, to me.
How about "argument " +String Int2String(ARG_IDX) +String " can't be converted to an unsigned i64"

Reply:
no. this error msg is hardcoded in the implementation of elrond. They will check this error msg in the output field. e.g., https://github.com/ElrondNetwork/elrond-wasm-rs/blob/bf438afe8216af459538565731ec8c76bf987d90/contracts/feature-tests/basic-features/mandos/echo_i32.scen.json#L111

Reply:
In the go implementation, the error msg is hard coded here https://github.com/ElrondNetwork/arwen-wasm-vm/blob/44d2b68c16a04266443f9dffc6ef21bda0b5d3cf/arwen/elrondapi/smallIntOps.go#L122